src/Chapter4.hs

```diff
@@ -489,6 +504,7 @@ Implement the 'Applicative' instance for our 'List' type.
   type.
 -}
 
+-- TODO
```

Comment:
Hey, do you need some help with this one? 🙂

Reply:
I think no, thank you! 
I will return to it later.